### PR TITLE
Update compiler error 0034 to use new format. #35959 modified

### PR DESCRIPTION
**Pull request by athulappadan**
Part of #35233
Addresses #35205

I have updated the unit test to check for the labels and also removed the tab characters. As I am relatively new to contributing to open source projects, please make suggestions if I am not using git in the correct way.

r? @jonathandturner

### CODE REVIEW

File: src/librustc_typeck/check/method/suggest.rs [hunk 1]
```diff
@@ -241,7 +241,8 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
 
             MethodError::Ambiguity(sources) => {
                 let mut err = struct_span_err!(self.sess(), span, E0034,
-                                               "multiple applicable items in scope");
+                                 "multiple applicable items in scope");
```
Comment (GuillaumeGomez) [inline src/librustc_typeck/check/method/suggest.rs]: One changing the indent?

Comment (athulappadan) [inline src/librustc_typeck/check/method/suggest.rs]: Oh!, I did that unintentionally. Will recommit it.

Comment (GuillaumeGomez) [inline src/librustc_typeck/check/method/suggest.rs]: You can commit then squash. It'll be faster. If you don't know how to squash, take a look [here](https://blog.guillaume-gomez.fr/Git-tips/1/2).

Comment (athulappadan) [inline src/librustc_typeck/check/method/suggest.rs]: Sorry, that blog link is not loading.

Comment (GuillaumeGomez) [inline src/librustc_typeck/check/method/suggest.rs]: Works fine for me. :-/

**Comment by athulappadan** [comment]
Any more changes to be done..?

r? @jonathandturner

**Comment by jonathandturner** [comment]
@athulappadan - looks pretty close.  Can you squash your git commits down from 3 commits to 1?  You can read how with articles like http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by athulappadan** [comment]
I am getting all kinds of errors while trying to squash and I am not able to figure it out myself. Seems like I have messed it up with my commits and merges. If possible, shall I open a fresh PR and finish the thing in one commit? (I think I need to spend some time to get a fix on how to use git properly)

r? @jonathandturner

**Comment by GuillaumeGomez** [comment]
:-/

Go ask on IRC if possible. We can't merge like this.

**Comment by jonathandturner** [comment]
@athulappadan - yes, you can close this PR and open a new one if you need to.  Getting used to squashing can take a little practice.

Something I like to do is to always use a branch for a specific PR.  That way, if I ever need to update something, I can switch back to `master`, `git pull` in master, and then rebase into my branch.  Since I rebase regularly, my changes are always at the top.  When I want to squash, I can `git log` to get first commit in the list that isn't mine, `git reset` to that commit number, and then add everything back all at once.

Not sure if that helps you.  It took me a while to get use to git and the techniques they use here, but after a while it gets easier.

**Comment by athulappadan** [comment]
Earlier itself I thought of making a new branch but then I saw the line saying push against master branch. Now I will make a new branch and push the changes.

**Pull request closed without merging**